charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -281,9 +281,7 @@ spec:
           - /proxy-server
         args:
         - --uds-name=/etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
-        - --log-file=/var/log/konnectivity-server/info.log
```

Comment:
The `/var/log/konnectivity-server` mount can be removed now right?

Reply:
Nice catch, thanks!